bot/menu.py

```diff
@@ -49,8 +49,22 @@ async def callback(self, interaction: Interaction):
             The action implemented by the user that needs to be notified.
             In the context of the bot, the action is a slash command
         """
+        prepend_message_string = ""
+        if self.is_adding_subscriber:
+            prepend_message_string = "You are now subscribed to "
+        else:
+            prepend_message_string = "You are now unsubscribed from "
+        converted_menu_values = list(map(int, self.values))
+        semantic_selected_tags = [
+            option.emoji.name + " " + option.label if option.emoji else option.label
+            for option in self.options
+            if option.value in converted_menu_values
+        ]
+        for tag in semantic_selected_tags:
+            prepend_message_string += tag + ", "
+        prepend_message_string = prepend_message_string[:-2]

```

Comment:
can't we just do something like `[tag in semantic_selected_tags].join(", ")` does this produce a string as i would expect it to?

Reply:
you can use join to append an iterable object, I replaced like so:
`response_message += ",".join(formatted_selected_tags)` where "," is a separator

Reply:
did you want to preserve the space though? 🤔

Reply:
Nvm i see that you did 👌🏻